pkg/kafka/kafka_util.go

```diff
@@ -18,16 +20,21 @@ var (
 	validSearchRegexp = regexp.MustCompile(`^([a-zA-Z0-9-_%]*[a-zA-Z0-9-_%])?$`)
 )
 
+type Validator struct {
+	Localizer  localize.Localizer
+	Connection factory.ConnectionFunc
+}
+
 // ValidateName validates the proposed name of a Kafka instance
-func ValidateName(val interface{}) error {
+func (v Validator) ValidateName(val interface{}) error {
```

Comment:
```suggestion
func (v *Validator) ValidateName(val interface{}) error {
```

it should use a pointer preferably.